docs/components/best-practices/architecture/deciding-about-your-stack.md

```diff
@@ -9,83 +9,77 @@ tags:
+## Further References
 
-|                                                               | Camunda Modeler                                                             | Third-Party Modeler (BPMN Standard Compliant)                                                                                                                               | Third-Party Modeler (Non-Compliant to Standard) |
-| ------------------------------------------------------------- | --------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------- |
-| Roundtrip in between process analysts and developers possible | &#10004;                                                                    | &#10004; (Carefully check level of BPMN compliance - the [Model Interchange Working Group](http://bpmn-miwg.github.io/bpmn-miwg-tools/) can serve as a first starting point |                                                 |
-| Use for process analysts                                      | &#10004;                                                                    | &#10004;                                                                                                                                                                    |                                                 |
-| Use for process developers                                    | &#10004;                                                                    |                                                                                                                                                                             |                                                 |
-| Use when                                                      | You do not have a BPMN standard compliant modeling tool already rolled out. | You already rolled out a BPMN tool with a standard compliancy sufficient for roundtrip.                                                                                     | Try to avoid                                    |
-|                                                               | [Download](https://camunda.org/download/modeler/)                           | [e.g. Cawemo](http://cawemo.com/)                                                                                                                                           |                                                 |
+Find more information in the [Camunda Platform 8 Documentation](https://docs.camunda.io/), or join the [Camunda Platform 8 Forum](https://forum.camunda.io/) and [Camunda Platform 8 Slack](https://camunda-cloud.slack.com/) community.
```

Comment:
This section seems redundant, and our Slack doesn't exist anymore. We can link to the contact page, but we are already in the docs.

Reply:
Okay, happy for removal